OtripleS.Web.Api/Controllers/SemesterCourseController.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
```

Comment:
**CopyRights** text needs to be inserted

Reply:
already inserted in second Commit 

Copyright notice added in SemseterCourse Controller

Reply:
Ok Thanks